cves/CVE-2009-2693.yml

```diff
@@ -43,15 +43,15 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: Vulnerability in directory pathing system. Allowed remote users to create and overwrite files without proper permission when using ".." in directory path. 
```

Comment:
How is it allowing users to create and overwrite files in this case? Are there any faulty implementations that may introduce to this vulnerability? What do you mean by proper permission? What permissions are by-passed? Please add some more description details, please.